File: C_PROGRAM/primenumber.c
#include<stdio.h>
int prime_number(int);
int main()
{
	int num;
	printf("enter  number:");
	scanf("%d",&num);
	int (*fp)(int)=prime_number;
	int a=fp(num);
	if(a==0)
		printf("given number is prime number:%d\n",num);
	else
		printf("given number is not a prime number:%d\n",num);
}
int prime_number(int number)
{
	int flag=0,i;
	for(i=2;i<=number/2;i++)
	{
		if(number%i==0)
		{
			flag=1;
			break;
		}
	}
	return flag;
}

File: python/primenumber.py
number=int(input("enter a number:"))
flag=0
for i in range(2,(number//2)+1):
    if(number%i)==0:
        flag=1
        break
    else:
        flag=0
if flag==1:
    print(number,"is not a prime number")
else:
    print(number,"is a prime number")
